### PR TITLE
[RUH-464] base microsite theme

### PR DESCRIPTION
**Pull request by bydawen**
https://youtrack.raccoongang.com/issue/ruh-164

### CODE REVIEW

File: lms/static/sass/blocks/_footer.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	color: #fff;
 	margin: 0 !important;
 	box-shadow: none !important;
-	border-top: none !important;
+	border-top: 1px solid rgba(255,255,255,0.1);
```
Comment (vlasovmichael) [inline lms/static/sass/blocks/_footer.scss]: stylelint

File: lms/static/sass/blocks/_header.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .global-header {
-	box-shadow: 0 0 6px 0 rgba(0, 0, 0, 0.75);
+	box-shadow: none;
 	padding: 0;
-	border-bottom: none;
+	border-bottom: 1px solid rgba(255,255,255,0.1);
```
Comment (vlasovmichael) [inline lms/static/sass/blocks/_header.scss]: stylelint

File: lms/static/sass/blocks/_header.scss [hunk 3]
```diff
@@ -526,6 +526,8 @@ div.mobile-menu {
 section.home {
 	> header,
 	.slider {
+		border-bottom: 1px solid rgba(255,255,255,0.1);
```
Comment (vlasovmichael) [inline lms/static/sass/blocks/_header.scss]: stylelint

File: lms/static/sass/blocks/_footer.scss (hunk 1 above)
Comment (bydawen) [reply lms/static/sass/blocks/_footer.scss]: fixed

File: lms/static/sass/blocks/_header.scss (hunk 2 above)
Comment (bydawen) [reply lms/static/sass/blocks/_header.scss]: fixed

File: lms/static/sass/blocks/_header.scss (hunk 3 above)
Comment (bydawen) [reply lms/static/sass/blocks/_header.scss]: fixed

**Pull request merged**